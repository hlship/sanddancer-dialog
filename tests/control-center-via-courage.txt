%% Roof to control center via courage
(setup flashlight)
teleport roof
u
(now) (knock holds #courage)
u
d
u

