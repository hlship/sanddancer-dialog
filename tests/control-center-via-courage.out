

Middle of Nowhere (in the pickup truck)
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air. On the dash you
can see a piece of jade. The glove compartment is closed.

> teleport roof

Roof
From the roof of the building, you can see the huge electrical tower rising up
in front of you. Some metal rungs once led to a control room higher up the
tower, but the lowest ones are well above your head. You'd have to jump out into
space to grab the bottom rung. A huge metal barrel stands upright and intact
nearby, large enough that you could climb up on it. At your feet is the way back
down.

> u
The metal rungs have rusted away and crumbled; the lowest are well above your
head. You don't have the courage to make the jump.

> (now) (knock holds #courage)
> u
Query succeeded: (get input [u])
You gather your courage and leap off the edge of the roof. Your hands grasp the
metal of the bottom-most rung, scraping and sliding, but holding. You pull
yourself up the rungs into the control center. Hey, it wasn't that hard after
all.

Control Center
Some kind of electrical control stuff must have happened here once, but it's
totally trashed now. Like, piles of trash, broken equipment, and junk are
everywhere.

But hey, on one pile there's a perfectly good can opener.

Back down is the only place to go.

> d
You climb down.

Roof
From the roof of the building, you can see the huge electrical tower rising up
in front of you. Some metal rungs once led to a control room higher up the
tower, but the lowest ones are well above your head. You'd have to jump out into
space to grab the bottom rung. A huge metal barrel stands upright and intact
nearby, large enough that you could climb up on it. At your feet is the way back
down.

> u
You pull yourself up the rungs into the control center. Hey, it wasn't that hard
after all.

Control Center
Some kind of electrical control stuff must have happened here once, but it's
totally trashed now. Like, piles of trash, broken equipment, and junk are
everywhere.

But hey, on one pile there's a perfectly good can opener.

Back down is the only place to go.

> 
